cpp/include/cudf/scalar/scalar_factories.hpp

```diff
@@ -56,6 +56,23 @@ std::unique_ptr<scalar> make_timestamp_scalar(
     cudaStream_t stream = 0,
     rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+/**
+ * @brief Construct scalar with uninitialized storage to hold a value of the
+ * specified timestamp `data_type` and a validity bool.
```

Comment:
```suggestion
 * specified fixed-width `data_type`.
```
This comment (as well as line 44 above) refers to a validity bool, but I don't see one in the signature; should that comment be removed, or maybe the parameter should be added?